Tools/parametric_model/generate_parametric_model.py

```diff
@@ -58,6 +59,8 @@ def str2bool(v):
 def start_model_estimation(config, log_path, data_selection=False, plot=False):
     print("Visual Data selection enabled: ", data_selection)
 
+    auto_data_selection=False
```

Comment:
Could we enable or disable this flag using the program args as it is done for example for `data_selection` or `plot`?

Reply:
Since it currently is not reliable i did not want to expose it to the user yet.

Reply:
makes sense